starter-code/css/main.css

```diff
@@ -1,8 +1,136 @@
 /*
 Colors:
-
 Text: 1A1A1A
 Green: #00B172
 White: #FFF
-
 */
+
+* {
+  font-family: "Montserrat", sans-serif;
+}
+
+h1 {
+  font-size: 80px;
+}
+
+h2 {
+  font-size: 40px;
+}
+
+p {
+  font-size: 25px;
+  font-weight: 100;
+}
+
+h3 {
+  text-decoration: underline;
+  font-size: 50px;
+  text-underline-position: below;
+}
+
+.navbar {
+  display: flex;
+  padding: 10px;
+  align-items: center;
+  justify-content: space-between;
+  font-size: 20px;
+}
+
+.logo {
+  width: 200px;
+}
+
+.links {
+  display: flex;
+  margin-left: 20px;
+}
+
+.intro {
+  background-image: url(../images/landing.jpg);
```

Comment:
include background-size: contain here in order to make the landing page image show its entire image.